manifests/m/Microsoft/PowerBI/2.118.828.0/Microsoft.PowerBI.installer.yaml

```diff
@@ -22,6 +22,6 @@ UpgradeBehavior: install
 Installers:
 - Architecture: x64
   InstallerUrl: https://download.microsoft.com/download/8/8/0/880BCA75-79DD-466A-927D-1ABF1F5454B0/PBIDesktopSetup_x64.exe
-  InstallerSha256: 14272EA7454690D77845562091408302835DA50FE6D5AF8D32CCB77C4A0D9EE7
+  InstallerSha256: 8A9F1B4549C36D7882F2E9EF1C91788BC4C58C24FDA9D8F320809D22F505F236
```

Comment:
```suggestion
  InstallerSha256: 14272EA7454690D77845562091408302835DA50FE6D5AF8D32CCB77C4A0D9EE7
```

Reply:
Was it reverted?

Reply:
@stephengillie - That is the hash of the old version. The issue here is with the PowerBI CDN, and is known. It takes many hours for all of their CDN endpoints to receive the new version, so I'd anticipate that the hash issue will resolve itself over the weekend